contracts/test/walletAction-test.ts

```diff
@@ -176,6 +180,56 @@ describe("WalletActions", async function () {
     );
   });
 
+  it("should send ETH with function call via fallback expander", async function () {
+    // send money to sender bls wallet
+    const sendWallet = await fx.lazyBlsWallets[0]();
+    const ethToTransfer = parseEther("0.001");
+    await fx.signers[0].sendTransaction({
+      to: sendWallet.address,
+      value: ethToTransfer,
+    });
+
+    const MockAuction = await ethers.getContractFactory("MockAuction");
+    const mockAuction = await MockAuction.deploy();
+    await mockAuction.deployed();
+
+    expect(await fx.provider.getBalance(sendWallet.address)).to.equal(
+      ethToTransfer,
+    );
+    expect(await fx.provider.getBalance(mockAuction.address)).to.equal(0);
+
+    const bundle = await sendWallet.signWithGasEstimate({
+      nonce: BigNumber.from(1),
+      actions: [
+        {
+          ethValue: ethToTransfer,
+          contractAddress: mockAuction.address,
+          encodedFunction: mockAuction.interface.encodeFunctionData("buyItNow"),
+        },
+      ],
+    });
+
+    await (
+      await fx.blsExpanderDelegator.run(
+        bundleCompressedOperations(
+          [
+            compressAsFallback(
+              Fixture.expanderIndexes.fallback,
+              bundle.senderPublicKeys[0],
+              bundle.operations[0],
+            ),
+          ],
+          bundle.signature,
+        ),
+      )
+    ).wait();
+
+    expect(await fx.provider.getBalance(sendWallet.address)).to.equal(0);
+    expect(await fx.provider.getBalance(mockAuction.address)).to.equal(
+      ethToTransfer,
+    );
+  });
```

Comment:
```suggestion
    await expect(fx.provider.getBalance(sendWallet.address)).to.eventually.equal(0);
    await expect(fx.provider.getBalance(mockAuction.address)).to.eventually.equal(
      ethToTransfer,
    );
  });
```

Reply:
Love it. Let's do it everywhere: https://github.com/web3well/bls-wallet/pull/518.